andela/the_spinners-frontend
# #169817520 Most travelled

#### What does this PR do?
Get Most traveled location
#### Description of Task to be completed?
- Users should be able to get info on the most frequently traveled center/destination
#### How should this be manually tested?
[login on barefoot](https://spinners-frontend-stage.herokuapp.com/login) then on your dashboard you should see most traveled in a graph.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[ #169817520](https://www.pivotaltracker.com/story/show/169817520)
#### Screenshots (if appropriate)
<img width="766" alt="Screen Shot 2020-03-30 at 20 12 37" src="https://user-images.githubusercontent.com/52409462/77946593-da4e2e00-72c2-11ea-8888-acf914b30c07.png">

#### Questions:
